.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: "CI"
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 7 * * *"
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.OS }}-latest
+    name: "tests"
+    strategy:
+      matrix:
+        PYTHON:
+          - 3.8
+          - 3.9
+        OS:
+          - ubuntu
+          - windows
+          - macos
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
More of a personal flair thing, but I'd add some empty lines between steps for readability.